openwisp_controller/config/tests/test_api.py

```diff
@@ -249,6 +252,32 @@ def test_device_list_api(self):
         with self.assertNumQueries(4):
             r = self.client.get(path)
         self.assertEqual(r.status_code, 200)
+        with self.subTest("should return devices ordered by most recent first"):
+            Device.objects.all().delete()
```

Comment:
Why is this needed? We could avoid it and add only 2 more devices so we end up with 3.